src/Microsoft.AspNetCore.Build/targets/ProductInfo.props

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <PropertyGroup>
-        <!-- Product-specific properties -->
-        <DefaultAuthors>Microsoft Open Technologies, Inc.</DefaultAuthors>
-        <ProductStartDate>2015-01-01</ProductStartDate>
-        <BuildQuality>alpha</BuildQuality>
-        <BuildVersion>1.0.1</BuildVersion>
-    </PropertyGroup>
+  <PropertyGroup>
+    <!-- Product-specific properties -->
+    <DefaultAuthors>Microsoft Open Technologies, Inc.</DefaultAuthors>
```

Comment:
KoreBuild 1.0 had "Microsoft Open Technologies, Inc." is that incorrect? @Eilon ?